draft-ietf-quic-transport.md

```diff
@@ -2803,6 +2810,46 @@ encoded as a single byte with the value 0x01.  An endpoint MAY treat the receipt
 of a frame type that uses a longer encoding than necessary as a connection error
 of type PROTOCOL_VIOLATION.
 
+## Latency Spin Bit {#spin-bit}
+
+The latency spin bit enables latency monitoring from observation points on the
+network path throughout the duration of a connection. Since it is possible to
+measure handshake RTT without a spin bit, it is sufficient to include the spin
```

Comment:
if there is no previous mentioning of any formula on how to measure the RTT at handshake, then couple of lines explaining that would make the reader wiser and interested.